packages/manager/apps/dedicated/client/app/ip/ip/ip-ip.controller.js

```diff
@@ -62,6 +63,26 @@ export default /* @ngInject */
     $scope.services = [];
     $scope.selectedService = null;
     $scope.serviceName = $stateParams.serviceName || '_ALL';
+    $scope.tracking = {
+      'enable-permanent-mitigation':
+        'dedicated::ip::dashboard::enable-permanent-mitigation:',
+      'create-firewall': 'dedicated::ip::dashboard::create-firewall',
+      'enable-firewall': 'dedicated::ip::dashboard::enable-firewall',
+      'ip::firewall': 'dedicated::ip::dashboard::ip::firewall',
+      'disable-firewall': 'dedicated::ip::dashboard::disable-firewall',
+      'mac-add': 'dedicated::ip::dashboard::mac-add',
+      statistics: 'dedicated::ip::dashboard::statistics',
+      'switch-auto-mitigation':
+        'dedicated::ip::dashboard::switch-auto-mitigation',
+      'move-failover': 'dedicated::ip::dashboard::move-failover',
+      'delete-failover': 'dedicated::ip::dashboard::delete-failover',
+      'mac-details': 'dedicated::ip::dashboard::mac-details',
+      'mac-delete': 'dedicated::ip::dashboard::mac-delete',
+      'ip::firewall::add-rule':
+        'dedicated::ip::dashboard::ip::firewall::add-rule',
+      'ip::firewall::delete-rule':
+        'dedicated::ip::dashboard::ip::firewall::delete-rule',
+    };
```

Comment:
**suggestion:** Common part of the label can be maintained as a different constant.

Reply:
Okay Anoop